version.go

```diff
@@ -140,3 +141,34 @@ func (self *VersionCheck) Verify() (bool, error) {
 	}
 	return true, nil
 }
+
+type OSVersionInfo struct {
+	MajorVersion     uint
+	MinorVersion     uint
+	BuildNumber      uint
+	PlatformId       VerPlatform
+	ServicePackName  string
+	ServicePackMajor uint
+	ServicePackMinor uint
+	SuiteMask        VerSuite
+	ProductType      VerProductType
+}
+
+func GetVersion() (OSVersionInfo, error) {
```

Comment:
`GetVersion` is the name of a Win32 function, even if it's one that we don't wrap.  I'd prefer that we avoid using it so that there aren't name conflicts if someone imports both the `gowin32` and `wrappers` packages into the global namespace.  Maybe something like `GetWindowsVersion` might be a good name.